ComfyUI/tests-unit/comfy_api_nodes_test/pika_frame_test.py

```
@@ -0,0 +1,197 @@
+        description = PikaFrameNode.DESCRIPTION
+        assert "motion_strength" in description
+        assert "future" in description
+
+    def test_minimum_duration_used(self):
+        """Test that minimum duration (5) is used in API request."""
+        with patch('comfy_api_nodes.nodes_pika.SynchronousOperation') as mock_sync:
+            with patch('comfy_api_nodes.nodes_pika.PollingOperation'):
+                with patch.object(self.node, '_extract_single_frame_to_png'):
+                    # Mock to capture the request data
+                    mock_sync.return_value.execute.return_value = Mock(video_id="test")
+                    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)